develop/plone/searching_and_indexing/query.rst

```diff
@@ -10,18 +10,17 @@ Querying
 Introduction
 ============
 
-*Querying* is the action to retrieve data from search indexes.  In Plone's
-case this usually means querying content items using the ``portal_catalog``
-tool.  Plone uses the :doc:`portal_catalog </develop/plone/searching_and_indexing/catalog>`
-tool to perform most content-related queries. Special catalogs, like
-``reference_catalog``, exist, for specialized and optimized queries.
+*Querying* is the action to retrieve data from search indexes.
+In Plone's case this usually means querying content items using the ``portal_catalog`` tool.
+
+Plone uses the :doc:`portal_catalog </develop/plone/searching_and_indexing/catalog>`
+tool to perform most content-related queries. Special catalogs, like ``reference_catalog``, exist, for specialized and optimized queries.
 
 
 Accesing the ``portal_catalog`` tool
 ====================================
 
-Plone queries are performed using ``portal_catalog`` persistent tool which
-is available as an persistent object at the site root.
+Plone queries are performed using ``portal_catalog`` persistent tool which is available as an persistent object at the site root.
```

Comment:
curious why we have to say it's persistent, and I think generally appending "tool" to portal_* is unnecessary